src/pages/learn/index.tsx

```diff
@@ -0,0 +1,943 @@
+                description="If you are new, start here. Learn the basics of crypto why Ethereum matters."
+              >
+                <>
+                  <CardImage>
+                    <GatsbyImage
+                      image={getImage(data.whatIsEth)}
+                      alt="Illustration of a person peering into a bazaar, meant to represent Ethereum.1"
+                    />
+                  </CardImage>
+                  <ButtonLink to="/what-is-ethereum/">
+                    What is Ethereum?
+                  </ButtonLink>
+                </>
+              </Card>
+              <Card
+                title="What is ETH?"
+                description="The currency of the Ethereum network is called ether or ETH."
+              >
+                <>
+                  <CardImage>
+                    <GatsbyImage
+                      image={getImage(data.eth)}
+                      alt=""
+                    />
+                  </CardImage>
+                  <ButtonLink to="/eth/">What is ETH?</ButtonLink>
+                </>
+              </Card>
+              <Card
+                title="Where can I get ETH?"
+                description="There are many ways to aquire ETH depending on your location."
+              >
+                <>
+                  <CardImage>
+                    <GatsbyImage
+                      image={getImage(data.impact)}
+                      alt=""
+                    />
+                  </CardImage>
+                  <ButtonLink to="/get-eth/">Where can I get ETH?</ButtonLink>
+                </>
+              </Card>
+            </CardGrid>
+
+            <AdditionalReadingHeader>
+              Additional reading on Ethereum
+            </AdditionalReadingHeader>
+            <DocsContainer>
+              <DocLink to="/smart-contracts/">
+                What are smart contracts?
+              </DocLink>
+              <DocLink to="/developers/docs//intro-to-ethereum/">
+                A technical introduction to Ethereum
+              </DocLink>
+              <DocLink to="/web3/">What is Web3?</DocLink>
+              <DocLink
+                to="https://www.youtube.com/watch?v=WSN5BaCzsbo"
+                isExternal
+              >
+                Decentralizing Everything
+              </DocLink>
+            </DocsContainer>
+          </Section>
+
+          <Section>
+            <h2 id={tocItems[1].id}>{tocItems[1].title}</h2>
+            <p>
+              It’s actually quite simple once you get the hang of it. You need
+              an application that is commonly called a “wallet”. It's an app
+              that helps you store your funds and to authenticate and interact
+              with applications on Ethereum.
+            </p>
+            <CardGrid>
+              <Card
+                title="What is a wallet?"
+                description="How you manage your ETH and your Ethereum accounts."
```

Comment:
Awkward phrasing here. I think explaining wallets in terms of accounts here is confusing/a big mental model shift that users struggle with. 

What about this:

```suggestion
                description="Wallets let you manage your ETH and Ethereum applications."
```